Modix.Services/Moderation/ModerationService.cs

```diff
@@ -628,8 +628,11 @@ private async Task<IEnumerable<GuildRoleBrief>> GetRankRolesAsync()
                 }))
                 .Select(r => r.Role);
 
-        private async Task RequireSubjectRankLowerThanModeratorRank(ulong guildId, ulong subjectId)
+        private async Task RequireSubjectRankLowerThanModeratorRankAsync(ulong guildId, ulong subjectId)
         {
+            if (AuthorizationService.CurrentUserIgnoresAuthorization.Value)
```

Comment:
See above - probably doesn't need to be nullable. Also, you should call `AuthorizationService.RequireAuthenticatedUser();` right before (apologies if it already gets called somewhere else in the stack).